styles.css

```diff
@@ -748,6 +824,195 @@ form {
   display: none;
 }
 
+/* Modal CSS */
+
+.modal {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  position: absolute;
+  width: 343px;
+  height: 718px;
+  left: calc(50% - 343px / 2);
+  top: 60px;
+}
```

Comment:
On the mobile version, I can only see the modal window when I click on the first project, none of the other cards display a modal window. Please fix that.

Additionally, the modal window is not properly displayed on the page.

<img width="226" alt="Screen Shot 2022-07-20 at 17 00 48" src="https://user-images.githubusercontent.com/82426682/180090045-ced136d4-d42b-454b-8145-8a3785481782.png">

<img width="187" alt="Screen Shot 2022-07-20 at 17 01 09" src="https://user-images.githubusercontent.com/82426682/180090090-aec3ff01-0b0c-4f9a-9956-50d3e9b901c9.png">

You need to make sure the background is blurred just like in the Figma design and that, as long as the modal window is open, the user can only see the modal window's content even if they scroll up and down the page.